Java/other/max_min_sum.java

```diff
@@ -0,0 +1,49 @@
+/*Given five positive integers, find the minimum and maximum values 
+that can be calculated by summing exactly four of the five integers. 
+Then print the respective minimum and maximum values as a single line of two space-separated long integers.*/
+import java.io.*;
+
+import java.util.*;//impoting java packages
+
+public class sol{
```

Comment:
Class name can be improved

Reply:
Okay, changing it to max_min_sum. Thanks for reviewing :)

Reply:
Class names in Java are mostly PascalCased.

Reply:
Okay, doing that.